_data/pii.yml

```diff
@@ -59,18 +56,20 @@ data:
       reason: Generated as a result of your use of the system
       handling: |
           We store the data securely in our database which only the
-          UseGalaxy.eu administrators have access to (BG, SJ, HR)
+          UseGalaxy.eu administrators have access to.
+
+          We perform automated security scans on job data to protect our systems against
```

Comment:
Is it important that we only do that for a subset of tools?

Reply:
We can specify it, but I think that would only limit us and I would assume we would need to specify which tools exactly and maybe warn the users in the toolbox directly. (but just guessing)